File: quests/poearthb/#Birak_Spawner.pl
sub EVENT_SPAWN {
	quest::settimer("depop",59);
}

sub EVENT_TIMER {
    $check_boss = $entity_list->GetMobByNpcTypeID(222001);#stonefist_clansman
	
    if (!$check_boss && !defined $qglobals{poeb_Birak} && !defined $qglobals{poeb_warlord}) {
		quest::spawn2(222017,0,0,$x,$y,$z,$h);  
        quest::stoptimer("depop");
        quest::depop();
	}
}

File: quests/poearthb/#Birak_Spawner.lua
function event_spawn(e)
	eq.set_timer("depop", 59000);
end

function event_timer(e)
	-- get quest globals
	local qglobals = eq.get_qglobals();

	-- get the list of NPCs in the zone
   local entity_list = eq.get_entity_list();

	-- Check to see if the Stonefist Clansman are up
	local check_stonefist = entity_list:GetMobByNpcTypeID(222001);

	if (not check_stonefist.valid and qglobals["poeb_Birak"] == nil and qglobals["poeb_warlord"] == nil ) then
		eq.spawn2(222017, 0, 0, e.self:GetX(), e.self:GetY(), e.self:GetZ(), e.self:GetHeading());
		eq.stop_timer("depop");
		eq.depop_with_timer();
	end
end
